.github/workflows/ccm.yml

```diff
@@ -6,6 +6,7 @@ on:
     branches: 
       - main
       - '[0-9][0-9][0-9][0-9].[0-9][0-9].*' # 2021.01.x
+      - '2024-03-01-dep-update'
     tags:
       - '[0-9][0-9][0-9][0-9].[0-9][0-9].[0-9][0-9]' # 2021.01.01
  
```

Comment:
I am going to add this feature Branch for Build for now, when merging this main i will remove it